CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- String comparisons to constant time comparisons in auth middleware library. [#825](https://github.com/rokwire/rokwire-building-blocks-api/issues/825)
```

Comment:
This needs to be moved under the Security section. Please see https://keepachangelog.com/en/1.0.0/#how. Thanks.

Reply:
didn't know about that. changed.